modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/chemistry_for_ERP.dm

```diff
@@ -343,245 +280,170 @@ Haha! Kill me please.
+	required_temp = 600
 	mix_message = "The mixture rapidly condenses and darkens in color..."
 
 /datum/chemical_reaction/camphor
-	results = list(/datum/reagent/drug/camphor = 6)
+	results = list(/datum/reagent/drug/aphrodisiac/camphor = 6)
 	required_reagents = list(/datum/reagent/carbon = 2, /datum/reagent/hydrogen = 2, /datum/reagent/oxygen = 2, /datum/reagent/sulfur = 1)
 	required_temp = 400
-	mix_message = "The mixture boils off a yellow, smelly vapor..."//Sulfur burns off, leaving the camphor
+	mix_message = "The mixture boils off a yellow, smelly vapor..."
 
-/datum/chemical_reaction/pentacamphor //liquid equivalent of horny jail
-	results = list(/datum/reagent/drug/pentacamphor = 1)
-	required_reagents = list(/datum/reagent/drug/camphor = 5, /datum/reagent/acetone = 1)
+/datum/chemical_reaction/pentacamphor
+	results = list(/datum/reagent/drug/aphrodisiac/camphor/pentacamphor = 1)
+	required_reagents = list(/datum/reagent/drug/aphrodisiac/camphor = 5, /datum/reagent/acetone = 1)
 	required_temp = 500
 	mix_message = "The mixture thickens and heats up slighty..."
 
 /datum/chemical_reaction/cum
 	results = list(/datum/reagent/consumable/cum = 5)
 	required_reagents = list(/datum/reagent/blood = 2, /datum/reagent/consumable/milk = 2, /datum/reagent/consumable/salt = 1)
-	mix_message = "The mixture turns into a gooey, musky white liquid..."//Blood for the organicness, milk for the whiteness, and salt for the taste
+	mix_message = "The mixture turns into a gooey, musky white liquid..."
 
 /datum/chemical_reaction/breast_enlarger
-	results = list(/datum/reagent/breast_enlarger = 8)
-	required_reagents = list(/datum/reagent/medicine/salglu_solution = 1, /datum/reagent/consumable/milk = 1, /datum/reagent/medicine/c2/synthflesh = 2, /datum/reagent/silicon = 3, /datum/reagent/drug/crocin = 3)
+	results = list(/datum/reagent/drug/aphrodisiac/breast_enlarger = 8)
+	required_reagents = list(/datum/reagent/medicine/salglu_solution = 1, /datum/reagent/consumable/milk = 1, /datum/reagent/medicine/c2/synthflesh = 2, /datum/reagent/silicon = 3, /datum/reagent/drug/aphrodisiac/crocin = 3)
 	mix_message = "the reaction gives off a mist of milk."
 
 /datum/chemical_reaction/penis_enlarger
-	results = list(/datum/reagent/penis_enlarger = 8)
-	required_reagents = list(/datum/reagent/blood = 5, /datum/reagent/medicine/c2/synthflesh = 2, /datum/reagent/carbon = 2, /datum/reagent/drug/crocin = 2, /datum/reagent/medicine/salglu_solution = 1)
+	results = list(/datum/reagent/drug/aphrodisiac/penis_enlarger = 8)
+	required_reagents = list(/datum/reagent/blood = 5, /datum/reagent/medicine/c2/synthflesh = 2, /datum/reagent/carbon = 2, /datum/reagent/drug/aphrodisiac/crocin = 2, /datum/reagent/medicine/salglu_solution = 1)
 	mix_message = "the reaction gives off a mist of milk."
 
-///////////////////////
-///BOTTLES AND PILLS///
-///////////////////////
+///////////////////////////
+///	PREMADE CONTAINERS	///
+///////////////////////////
```

Comment:
Try to change these to, like,
```
/**
* PREMADE CONTAINERS
*/
```
I've been told it literally breaks autodocs, so... You never know.